pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_household_foi.py

```diff
@@ -17,29 +18,46 @@ def setUpClass(cls) -> None:
         infectee, both in the same place and household.
         """
         super(TestHouseholdInfection, cls).setUpClass()  # Sets up parameters
-        cls.cell = pe.Cell()
-        cls.microcell = pe.Microcell(cls.cell)
-        cls.infector = pe.Person(cls.microcell)
-        cls.infector.infectiousness = 1.0
-        cls.infectee = pe.Person(cls.microcell)
         cls.time = 1
+        cls._population = pe.Population()
+        cls._population.add_cells(1)
+        cls._population.cells[0].add_microcells(1)
+        cls._population.cells[0].microcells[0].add_people(2)
+        for person in cls._population.cells[0].microcells[0].persons:
+            person.update_status(InfectionStatus(7))
+            person.infectiousness = 1.0
+        cls.placeclosure = PlaceClosure(start_time=1,
```

Comment:
I don't think these parameters should be in the setUpClass. Make a seperate function ' test_house_place_closure'  as done for case isolation and put the parameters you want to check in there.